### DISCUSSION TITLE
ft-fms-docker for FMS 19.6.1.52

### DISCUSSION THREAD

**Discussion by kietnt072** [fmgarage/ft-fms-docker#15] [Ideas]
Since Apple released FMS 19.6.1.52 only runs on Ubuntu 20.04, your package could not build the Docker Image and Docker Container.
If you have time, please build the new version for your Package.

Thanks

**Comment by sk-gara** [comment]
I will have a look at this. If we're lucky, we'll only have to switch to the Ubuntu 20.04 base image.

**Comment by sk-gara** [comment]
Doesn't look so good. Installation fails with 19.5 and 19.6 when using an Ubuntu:20.04 base image. Using our script _and_ also when following Claris' Tutorial at [https://support.claris.com/s/article/Running-FileMaker-Server-in-a-Docker-container-for-Ubuntu-20-04?language=en_US](https://support.claris.com/s/article/Running-FileMaker-Server-in-a-Docker-container-for-Ubuntu-20-04?language=en_US).

This is the error:
```
Unpacking filemaker-server (19.5.1.43) ...
dpkg: error processing archive /install/filemaker-server-19.5.1.43-amd64.deb (--unpack):
 unable to securely remove '/opt/FileMaker/FileMaker Server/Data/Caches.dpkg-new': Permission denied
dpkg-deb: error: paste subprocess was killed by signal (Broken pipe)
```

**Comment by sk-gara** [comment]
What filesystem have you set in Docker Desktop settings?

**Comment by kietnt072** [comment]
I used Bind mount and volume as your previous version via the install.sh
and helper_ubuntu.sh

**Comment by sk-gara** [comment]
Sorry, I meant in the Docker Desktop Dashboard, go to Settings/General and under "Choose file sharing implementation for your containers", you have 3 options: 
1. gRPC FUSE
2. VirtioFS
3. osxfs (Legacy)

You probably have VirtioFS activated. Please try setting it to gRPC FUSE, clean up everything (all images & containers).

Also - I mixed up this issue with #16 , sorry.
I will be pushing an update in a few minutes that will add an additional option in the install process. Please try and see if that solves the problem.

**Comment by kietnt072** [comment]
I will change the setting as you suggested and try again. Thanks
